modules/dists/StencilDist.chpl

```diff
@@ -1422,6 +1445,52 @@ iter StencilArr.dsiBoundaries(param tag : iterKind) where tag == iterKind.standa
   }
 }
 
+//
+// Returns a view of this Stencil-distributed array without any fluff.
+// Useful for ensuring that reads/writes always operate on non-cached data
+//
+// Copies the range slice function in ChapelArray.
+//
+proc _array.noFluffView() {
+  var a = _value.dsiNoFluffView();
+  a._arrAlias = _value;
+  pragma "dont disable remote value forwarding"
+  proc help() {
```

Comment:
A more descriptive name for this might be _helpful_. Maybe ` proc incRefCounts()`?